src/test/java/seedu/recipe/storage/jsonadapters/JsonAdaptedNameTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.recipe.storage.jsonadapters;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.recipe.testutil.Assert.assertThrows;
+import static seedu.recipe.testutil.TypicalRecipes.CACIO_E_PEPE;
+import static seedu.recipe.testutil.TypicalRecipes.CACIO_NAME;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.recipe.commons.exceptions.IllegalValueException;
+import seedu.recipe.model.recipe.Name;
+
+
+public class JsonAdaptedNameTest {
+
+    private static final String VALID_NAME = "Cheeseburger";
+    private static final String INVALID_NAME = "Burger123";
```

Comment:
Might wanna check if they changed the regex to make it more lenient @filbertphang @alson001

Reply:
Changed according to filbert changes.